### PR TITLE
prompt to create build

### PR DESCRIPTION
**Pull request by mattmanning**
screenshot

### CODE REVIEW

File: views/app.tmpl [hunk 1]
```diff
@@ -271,17 +271,25 @@
...
 				<tr>
-					<td class="statusicon">{{ statusicon .Status }}</td>
-					<td class="id">{{ .Id }}</td>
-					<td>{{ timeago .Started }}</td>
-					<td class="id">{{ .Release }}</td>
-					<td>{{ duration .Started .Ended }}</td>
-					<td class="buttons">
-						<button class="btn btn-xs btn-info build-logs" title="Build Logs" data-toggle="modal" data-target="#build-logs-{{ .Id }}">Build Logs</button>
+					<td colspan=6>
+						You don't have an builds yet. Click Build to create one.
```
Comment (ddollar) [inline views/app.tmpl]: "an" should be "any"

Comment (ddollar) [inline views/app.tmpl]: Perhaps say "Click the green Build button above to create one."

**Comment by mattmanning** [comment]
updated screenshot

**Comment by ddollar** [comment]
:+1:

**Comment by nzoschke** [comment]
:+1:

**Pull request merged**